packages/webapp/src/components/Task/PureIrrigationTask/index.jsx

```diff
@@ -68,13 +59,25 @@ export default function PureIrrigationTask({
 
   const dispatch = useDispatch();
 
+  const getDefaultIrrigationTypes = () => {
+    let  defaultIrrigationTaskTypes = [];
+    for (let type of irrigationTaskTypes) {
+      if (type.farm_id === null) {
+        defaultIrrigationTaskTypes.push(type.irrigation_type_name);
+      }
+    }
+    return defaultIrrigationTaskTypes;
+  }
+
   const IrrigationTypeOptions = useMemo(() => {
     let options;
+    let defaultIrrigationTaskTypes = getDefaultIrrigationTypes()
     options = irrigationTaskTypes.map((irrigationType) => {
       return {
         value: irrigationType.irrigation_type_name,
         label: defaultIrrigationTaskTypes.includes(irrigationType.irrigation_type_name)
           ? t(`ADD_TASK.IRRIGATION_VIEW.TYPE.${irrigationType.irrigation_type_name}`)
+          // is the false case here really needed?
```

Comment:
@craigyu 

I think yes, if the tasktype is included in the default then we need to translate to local language. If it is not a default type then the user will have named it appropriately to their own language.

Reply:
In the false case it's calling `t(irrigationType.irrigation_type_name)`, but we know there won't be any translation for user generated task name, so I'm not sure if calling the `t` function here is necessary since we could just display it as it is. It's working the way it is though.

Reply:
Oh I think I see what you are saying. Do you mean just the t() should be removed from the false case?  I am surprised that translation function would be working without a value for that haha Yeah I think you can remove that.

Reply:
haha yes it's basically relying on the `t()`'s fault tolerance ability

Reply:
Ok it's removed and tested working